### PR TITLE
APP-25979: feat: add date check for aws tags

### PR DESCRIPTION
**Pull request by clementlecorre**

### CODE REVIEW

File: commands/sns/command.go [hunk 1]
```diff
@@ -206,6 +233,33 @@ func snsTopicARNToTopicName(topicARN *string) *string {
...
+		if tag.Key != nil && *tag.Key == CheckTagNameUpdateDate {
+			if tag.Value == nil {
+				return false, nil
+			}
+			dataTime, err := time.Parse("2006-01-02T15:04:05.000-03:00", *tag.Value)
+			if err != nil {
+				return false, err
+			}
+			if dataTime.After(*rootArgs.UnusedSinceDate) {
+				l.Infof("%s is not old (%s) enough to delete", *topicArn, dataTime.String())
```
Comment (fclairamb) [inline commands/sns/command.go]: This is not how structured logging works.

```go
l.Infow("topic is not old enough to delete", "topicArn", *topicArn, "date", dataTime.String())
```

**Review by fclairamb — COMMENTED** [review]
LGTM

But please do use structured logging for real.

File: commands/sns/command.go (hunk 1 above)
Comment (clementlecorre) [reply commands/sns/command.go]: done! 👍

File: commands/sns/command.go [hunk 2]
```diff
@@ -8,11 +8,13 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"time"
 
 	"github.com/aws/aws-sdk-go/service/sns"
 	"github.com/habx/aws-mq-cleaner/flags"
 	"github.com/habx/aws-mq-cleaner/helpers"
 	"github.com/habx/aws-mq-cleaner/logger"
+	t "github.com/habx/aws-mq-cleaner/time"
```
Comment (jean9696) [inline commands/sns/command.go:17]:
```suggestion
	t"github.com/habx/aws-mq-cleaner/time"
```

Comment (jean9696) [inline commands/sns/command.go:17]:
```suggestion
	"github.com/habx/aws-mq-cleaner/time"
```

**Review by fclairamb — APPROVED** [review]

**Pull request merged**